### PR TITLE
update nosql-quiz.md

### PR DESCRIPTION
**Pull request by KeithDinh**
add 4 questions/answers and 1 question (48) without answer

### CODE REVIEW

File: nosql/nosql-quiz.md [hunk 1]
```diff
@@ -251,3 +251,33 @@
...
  - [ ] medium
  - [x] a single bit
+
+#### Q44. You need to generate a unique, sequential identifier for each value stored in a Redis cluster. What do you do?
+ - [ ] Implement a SortedSet object to generate a value.
+ - [ ] Use the GUID keyword to generate a value.
+ - [ ] Implement a List object to generate a value.
+ - [x] Use the INCR keyword to generate a value
+
+ #### Q45. You want to return a list of all elements in two Redis sets. Which keyword do you use?
```
Comment (Ebazhanov) [inline nosql/nosql-quiz.md:261]:
```suggestion
#### Q45. You want to return a list of all elements in two Redis sets. Which keyword do you use?
```

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
Good progress @KeithDinh! Thank you 👍 , but before we merge could you please fix conflicts between your branch and master, thank you in advance! 
image

**Comment by KeithDinh** [comment]
@Ebazhanov I resolved the conflict. Let me know if it goes wrong.

**Comment by Ebazhanov** [comment]
> @Ebazhanov I resolved the conflict. Let me know if it goes wrong.

Wow, so fast, thanks a lot will merge now!!!!! 👍

**Pull request merged**